src/bin/deku_node.ml

```diff
@@ -29,22 +29,6 @@ let handle_request (type req res)
     | Error err -> raise (Failure err) in
   Dream.post E.path handler
 
-(* POST /append-block-and-signature *)
```

Comment:
>Hmm, maybe because we plan to handle it with Pollinate msg_handler instead? Sorry for making comments before reading the whole PR, this is my form of active reading 😅

Yes, that's it! We will no longer receive/send messages over this endpoint. Pollinate will be used instead.